docs/internals/git.txt

```diff
@@ -54,12 +54,12 @@ website can be found at `github.com/django/djangoproject.com
 .. _branches: https://github.com/django/django/branches
 .. _tags: https://github.com/django/django/tags
 
-The master branch
-=================
+The main branch
+===============
 
```

Comment:
I think it would be useful to add a paragraph *somewhere* about the change, and this page seems logical.

Reply:
Suggested wording:

```
... note:

    Prior to March 2021, the main branch was called `master`. For more information see `PR #14048 <https://github.com/django/django/pull/14048>`__.
```